seacrowd/sea_datasets/copal/copal.py

```diff
@@ -0,0 +1,155 @@
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{wibowo2023copal,
+  title={COPAL-ID: Indonesian Language Reasoning with Local Culture and Nuances},
+  author={Wibowo, Haryo Akbarianto and Fuadi, Erland Hilman and Nityasya, Made Nindyatama and Prasojo, Radityo Eko and Aji, Alham Fikri},
+  journal={arXiv preprint arXiv:2311.01012},
+  year={2023}
+}
+"""
+_DATASETNAME = "COPAL"
```

Comment:
this one shd be okay if kept as lowercase, kinda weird to have this value as uppercase, even more this is being defined for URLS...